File: MODUL-2/python/PRAK202-2310817210007-RaymondHariyono.py
print("Input")
pertama=float(input("Masukkan Nilai Pertama:"))
kedua=float(input("Masukkan Nilai Kedua:"))

hasil= pertama + kedua

pertama = (  f"{int(pertama)}" if pertama.is_integer() else f"{pertama:.2f}")
print()
print("output")
print(f'Hasil dari penjumlahan nilai pertama "{pertama}" dan nilai kedua "{kedua}" adalah "{hasil:.2f}"')

File: MODUL-2/bahasa.c/PRAK202-2310817210007-RaymondHariyono.c
#include <stdio.h>

int main(){
    float pertama;
    float kedua;

    printf("Input \n");
    
    printf("Masukkan Nilai Pertama: ");
    scanf("%g",& pertama);
    
    printf("Masukkan Nilai Kedua: ");
    scanf("%f",& kedua);
   
    float hasil = pertama + kedua;
    
    printf("\n");
    printf("Output \n");
    printf("hasil dari nilai pertama \"%g\" dan nilai kedua \"%.1f\" adalah \"%.2f",pertama, kedua, hasil);
    
return 0;
}